spark/src/main/scala/org/apache/spark/sql/delta/util/FileNames.scala

```diff
@@ -42,18 +42,6 @@ object FileNames {
    */
   def unsafeDeltaFile(path: Path, version: Long): Path = new Path(path, f"$version%020d.json")
 
-  /**
-   * Returns the delta (json format) path for a given delta file.
-   * WARNING: This API is unsafe and deprecated. It can resolve to incorrect paths if the table has
-   * Managed Commits. It will be removed in future versions.
-   * Use DeltaCommitFileProvider(snapshot).deltaFile instead to guarantee accurate paths or
-   * unsafeDeltaFile for potentially incorrect file name resolution.
-   */
-  @deprecated(
-    "This method is deprecated and will be removed in future versions. " +
-      "Use DeltaCommitFileProvider(snapshot).deltaFile or unsafeDeltaFile instead")
-  def deltaFile(path: Path, version: Long): Path = unsafeDeltaFile(path, version)
```

Comment:
Lets get this reviewed by @tdas also.

Reply:
This is internal method. no users should be using this. this deprecation i guess is for delta devs, not users. So this should be good to remove.